docs-2.0/nebula-explorer/13.choose-graphspace.md

```diff
@@ -0,0 +1,23 @@
+# Choose graph spaces
+
+You must first choose a graph space and then query and analyze data with Explorer. This topic introduces how to choose a graph space.
+
+## Prerequisite
+
+You have connected to Explorer. For details, see [Connect to Explorer](deploy-connect/ex-ug-connect.md).
+
+## Steps
+
+After connecting to Explorer, the system automatically displays the graph space selection page. You only need to select the target graph space.
+
+![Create_graphspace](https://docs-cdn.nebula-graph.com.cn/figures/create-graphspace_en.png)
```

Comment:
I think this part would be enough:
![image](https://user-images.githubusercontent.com/69501902/160965503-b784f717-0c90-46f1-8e56-ac5e69649416.png)

We should let the users focus on the minimum area they need for carrying out the operations and save some space by using screenshots of accurate areas.

And highlighting the space card may also help users focus.